transforms/dec128.js

```diff
@@ -0,0 +1,78 @@
+const DECIMAL_LITERAL = "DecimalLiteral";
+
+const opToName = {
+  '+': 'add',
+  '*': 'mul',
+  '-': 'sub',
+  '/': 'div',
+};
+
+const includedOps = Object.keys(opToName);
+
+const isDecimalOrBinaryExpBare = (t) => (node) => {
+  return isDecimal(node) || t.isBinaryExpression(node);
+};
+
+const areMixedTypesBare = (isDecimalOrBinaryExp) => (a, b) => {
+
+  return (isDecimalOrBinaryExp(a) && !isDecimalOrBinaryExp(b))
+    || !(isDecimalOrBinaryExp(a) && isDecimalOrBinaryExp(b))
+};
+
+const isDecimal = (node) => {
+  return node.get('callee').isIdentifier({name: 'Decimal'}) || node.type === DECIMAL_LITERAL;
```

Comment:
If `node` is not a `CallExpression`, I think `node.get("callee")` would return null and thus throw when getting `.isIdentifier`. You should first check if `node.isCallExpression()`.

Reply:
I think this can be addressed by reversing the order of the check and using optional chaining, since if it isn't the Decimal primitive or a `CallExpression` (which is to say, a previously converted Decimal), this is correctly `false`. But it doesn't matter because we changed the implementation. 😆  

Related though: should I put adding an `isDecimal` check to Babel itself on my to-do list? I'm not sure what the preferred timing is here in terms of syntax and transforms.